roles/infrastructure/tasks/main.yml

```diff
@@ -15,5 +15,5 @@
 # limitations under the License.
 
 - include_tasks: validate.yml
-- include_tasks: initialize_setup.yml
-- include_tasks: setup.yml
+# - include_tasks: initialize_setup.yml
```

Comment:
Why is this commented out?

Reply:
Apologies, this is a hangover from when I first started this PR and trying to understand cloudera.exe. Uncommented these lines in [617d68c](https://github.com/cloudera-labs/cloudera.exe/pull/56/commits/617d68c30954838b9e3c921de9e84d92dc594445)